docs/content/doc/actions/comparison.en-us.md

```diff
@@ -0,0 +1,157 @@
+---
+date: "2023-04-27T15:00:00+08:00"
+title: "Compared to GitHub Actions"
+slug: "usage/actions/comparison"
+weight: 30
+draft: false
+toc: false
+menu:
+  sidebar:
+    parent: "actions"
+    name: "Comparison"
+    weight: 30
+    identifier: "actions-comparison"
+---
+
+# Compared to GitHub Actions
+
+Even though Gitea Actions is designed to be compatible with GitHub Actions, there are some differences between them.
+
+**Table of Contents**
+
+{{< toc >}}
+
+## Additional features
+
+### Absolute action URLs
```

Comment:
I think configurable actions site should also be mentioned here. That means, the default site is not github.com and configurable.

Reply:
Added in "Different behavior"

See 6024cfc04d